Set Warning to Debug when older cluster does not have resources

```
5:28PM INF >>> Kube No Trouble `kubent` <<<
5:28PM INF version nightly-0.4.0-83-g055df4e (git sha 055df4e213a394ad107021724714c5130759b284)
5:28PM INF Initializing collectors and retrieving data
5:28PM INF Target K8s version is 1.15+
5:28PM WRN Failed to retrieve: networking.k8s.io/v1, Resource=ingressclasses: the server could not find the requested resource
5:28PM WRN Failed to retrieve: storage.k8s.io/v1, Resource=csidrivers: the server could not find the requested resource
5:28PM WRN Failed to retrieve: storage.k8s.io/v1, Resource=csinodes: the server could not find the requested resource
5:28PM WRN Failed to retrieve: authorization.k8s.io/v1, Resource=subjectaccessreviews: the server does not allow this method on the requested resource
5:28PM WRN Failed to retrieve: authorization.k8s.io/v1, Resource=selfsubjectaccessreviews: the server does not allow this method on the requested resource
5:28PM WRN Failed to retrieve: authorization.k8s.io/v1, Resource=localsubjectaccessreviews: the server could not find the requested resource
5:28PM WRN Failed to retrieve: authentication.k8s.io/v1, Resource=tokenreviews: the server does not allow this method on the requested resource
5:28PM WRN Failed to retrieve: certificates.k8s.io/v1, Resource=certificatesigningrequests: the server could not find the requested resource
5:28PM WRN Failed to retrieve: apiextensions.k8s.io/v1, Resource=customresourcedefinitions: the server could not find the requested resource
5:28PM WRN Failed to retrieve: admissionregistration.k8s.io/v1, Resource=mutatingwebhookconfigurations: the server could not find the requested resource
5:28PM WRN Failed to retrieve: admissionregistration.k8s.io/v1, Resource=validatingwebhookconfigurations: the server could not find the requested resource
5:28PM WRN Failed to retrieve: node.k8s.io/v1, Resource=runtimeclasses: the server could not find the requested resource
5:28PM WRN Failed to retrieve: policy/v1, Resource=poddisruptionbudgets: the server could not find the requested resource
5:28PM WRN Failed to retrieve: discovery.k8s.io/v1, Resource=endpointslices: the server could not find the requested resource
5:28PM WRN Failed to retrieve: batch/v1, Resource=cronjobs: the server could not find the requested resource
5:28PM INF Retrieved 58 resources from collector name=Cluster
5:28PM INF Retrieved 939 resources from collector name="Helm v2"
5:28PM INF Retrieved 24 resources from collector name="Helm v2"
5:28PM INF Loaded ruleset name=custom.rego.tmpl
5:28PM INF Loaded ruleset name=deprecated-1-16.rego
5:28PM INF Loaded ruleset name=deprecated-1-22.rego
5:28PM INF Loaded ruleset name=deprecated-1-25.rego
panic: interface conversion: interface {} is nil, not string
```

--- a/pkg/collector/cluster.go
+++ b/pkg/collector/cluster.go
@@ -111,7 +111,7 @@ func (c *ClusterCollector) Get() ([]map[string]interface{}, error) {
 		log.Debug().Msgf("Retrieving: %s.%s.%s", g.Resource, g.Version, g.Group)
 		rs, err := ri.List(metav1.ListOptions{})
 		if err != nil {
-			log.Warn().Msgf("Failed to retrieve: %s: %s", g, err)
+			log.Debug().Msgf("Failed to retrieve: %s: %s", g, err)
 			continue
 		}
 

